plenum/persistence/req_id_to_txn.py

```diff
@@ -0,0 +1,50 @@
+from hashlib import sha256
+from typing import Optional
+
+from plenum.persistence.kv_store_leveldb import KVStoreLeveldb
+
+
+class ReqIdrToTxn:
+    """
+    Stores a map from client identifier, request id tuple to transaction
+    sequence number
+    """
+    def __init__(self, *args, **kwargs):
+        raise NotImplementedError
+
+    def add(self, identifier, reqId, seqNo):
+        raise NotImplementedError
+
+    def get(self, identifier, reqId):
+        raise NotImplementedError
+
+
+class ReqIdrToTxnKVStore(ReqIdrToTxn):
+    def __init__(self, dbPath):
```

Comment:
If we need to change a KVStore type, we need to change implementation. It's better to inject `db` into constructore (in addition to `dbpah`), so that using another DB (for example RocksDB) can be done easily using existing `ReqIdrToTxnKVStore` class.